rd_stats.h

```diff
@@ -187,6 +187,10 @@ struct stats_paging {
 	unsigned long pgscan_kswapd	__attribute__ ((aligned (8)));
 	unsigned long pgscan_direct	__attribute__ ((aligned (8)));
 	unsigned long pgsteal		__attribute__ ((aligned (8)));
+	unsigned long pgpromote_success __attribute__ ((aligned (8)));
+	unsigned long pgdemote_kswapd   __attribute__ ((aligned (8)));
+	unsigned long pgdemote_direct   __attribute__ ((aligned (8)));
+	unsigned long pgdemote_khugepaged __attribute__ ((aligned (8)));
```

Comment:
In fact we only display a single value for all the pages which have been demoted. So no need to save in the `saDD` datafile all three fields `pgdemote_kswapd`, `pgdemote_direct` and `pgdemote_hugepaged`. Add only one field named `pgdemote` (it will save space!) and compute its value when reading the `vmstat` file.

Reply:
@sysstat : I have finished the modification, thank you for your correction.